charts/operator-wandb/charts/mysql/templates/statefulset.yaml

```diff
@@ -49,12 +49,12 @@ spec:
               valueFrom:
                 secretKeyRef:
                   name: {{ include "wandb.mysql.passwordSecret" . }}
-                  key: MYSQL_ROOT_PASSWORD
+                  key: {{ if .Values.global.mysql.passwordSecret.rootKey }}{{ .Values.global.mysql.passwordSecret.rootKey }}{{ else }}MYSQL_ROOT_PASSWORD{{ end }}
```

Comment:
Since we default `.Values.global.mysql.passwordSecret.rootKey` to `MYSQL_ROOT_PASSWORD` I'm not sure the conditional is doing anything here